EU4toV2/Source/V2World/Province/Province.cpp

```diff
@@ -120,9 +122,16 @@ void V2::Province::convertFromOldProvince(const std::vector<std::shared_ptr<EU4:
 
 	// Single HRE province is enough
 	for (const auto& oldProvince: provinceSources)
+	{
 		if (oldProvince->inHre())
+		{
 			inHRE = true;
-
+			if (const auto& hreReforms = sourceWorld.getHREReforms();
+				 std::find(hreReforms.begin(), hreReforms.end(), "emperor_reichskrieg") == hreReforms.end())
```

Comment:
Also, it'd be better design to have the world cache the fact that the HRE has been decentralised, rather than directly looking up a particular reform (and doing so for each province).

Reply:
that.